plugins/examples/rescore/src/main/java/org/elasticsearch/example/rescore/ExampleRescoreBuilder.java

```diff
@@ -225,7 +223,7 @@ public Explanation explain(int topLevelDocId, IndexSearcher searcher, RescoreCon
         }
 
         @Override
-        public void extractTerms(IndexSearcher searcher, RescoreContext rescoreContext, Set<Term> termsSet) {
+        public void extractTerms(IndexSearcher searcher, RescoreContext rescoreContext) {
```

Comment:
The new signature is a bit weird, the only option is to call `createWeight` on the searcher but it's obfuscated so you need to check an actual implementation to realize that.